sites/geohub/src/lib/config/AppConfig/StacProducts.ts

```diff
@@ -0,0 +1,27 @@
+export const StacProducts = [
+	{
+		collection_id: 'sentinel-2-l2a',
+		products: [
+			{
+				name: 'NDVI',
+				label: 'Normalized Difference Vegetation Index',
+				expression: '(nir-red)/(nir+red)',
+				assets: ['red', 'nir']
+			},
+			{
+				name: 'NDWI',
+				label: 'Normalized Difference Water Index',
+				expression: '(green-nir)/(green+nir)',
+				assets: ['green', 'nir']
+			},
+			{
+				name: 'NDMI',
+				label: 'Normalized Difference Moisture Index',
+				expression: '(nir-swir)/(nir+swir)',
+				assets: ['nir', 'swir']
+			}
+		]
+	}
+];
+
+export const TestTitilerUrl = 'https://titiler.cyverse.org';
```

Comment:
I suggest move this info into new database tables

For example,

- `product` table
  - id: sequence or string (PK)
  - name: product name
  - expression: string
  - assets: JSON (string[])

- `stac_collection_product`
  - id: sequence
  - collection_id - FK to `stac` table with cuscade
  - product_id - FK to `product` table without cuscade

We also need new endpoints to get/post/put/delete product and link to collection ID